test/clangd-diagnostic-gutter.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# REQUIRES: command -v clangd
+
+. test/lib.sh
+
+cat >> .config/kak/kakrc << EOF
+set-option global lsp_diagnostic_line_error_sign   ' X'
+set-option global lsp_diagnostic_line_hint_sign    '¿ '
+set-option global lsp_diagnostic_line_info_sign    '¡ '
+set-option global lsp_diagnostic_line_warning_sign 'W '
+EOF
+
+cat > main.c << EOF
+void main(int argc, char** argv) {}
+syntax error
+EOF
+
+test_tmux_kak_start main.c
+test_tmux capture-pane -p | sed 2q
+# CHECK: {{W }}void main(int argc, char** argv) {}
+# CHECK: {{ X}}syntax error
+
+test_tmux send-keys %:comment-line Enter
+test_sleep
```

Comment:
I dropped this line, to make the test more resilient to unrelated changes (there's a double space in our modeline that I'd like to get rid of eventually).
Also "unit" test is probably the wrong term, it's more of an end-to-end system test because it tests interaction, plus it even depends on external components (clangd) outside our control (I plan to replace it with a fake language servers eventually).